src/docfx/restore/PackageResolver.cs

```diff
@@ -2,281 +2,18 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
-using System.IO;
 
 namespace Microsoft.Docs.Build
 {
-    internal class PackageResolver : IDisposable
+    internal abstract class PackageResolver : IDisposable
     {
-        // NOTE: This line assumes each build runs in a new process
-        private static readonly ConcurrentDictionary<(string gitRoot, string url, string branch), Lazy<PathString>> s_gitRepositories
-                          = new ConcurrentDictionary<(string gitRoot, string url, string branch), Lazy<PathString>>();
+        public abstract Package ResolvePackage2(PackagePath package, PackageFetchOptions options);
```

Comment:
>ResolvePackage2 [](start = 32, length = 15)

emmm, this name :/

Reply:
how about make this `ResolvePackage`, and rename the following `ResolvePackage` to `ResolvePackagePath`

---
In reply to: [486133240](https://github.com/dotnet/docfx/pull/6531#discussion_r486133240) [](ancestors = 486133240)